viewing_party/party.py

```diff
@@ -1,23 +1,175 @@
+# -----------------------------------------
 # ------------- WAVE 1 --------------------
+# -----------------------------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie = {}
+
+    if not title or not genre or not rating:
+        return None
+    else:
+        movie["title"] = title
+        movie["genre"] = genre
+        movie["rating"] = rating
+
+    return movie

```

Comment:
A common pattern we will see in our functions is to start with a guard clause and then follow up with the important logic we need to handle. In cases like this, consider moving the dictionary creation out of the else statement and letting it stay as the main logic of our function. 

Similarly, in cases like this where we are creating the dictionary once, consider just hardcoding it within the return statement. This might look something like:

```suggestion
    return {
         "title": title,
         "genre": genre,
         "rating": rating
    }

```